### PR TITLE
PSK-22 Browse screen FE

### PR DESCRIPTION
**Pull request by somkamarius**

### CODE REVIEW

**Comment by lamu6354** [comment]
Condition filter in BrowseScreen needs to be finished and getAll from ItemServices should be rewritten, since it breaks the single-responsibility principle.

**Review by lamu6354 — APPROVED** [review]
Condition filter in BrowseScreen needs to be finished and getAll from ItemServices should be rewritten, since it breaks the single-responsibility principle

**Review by NerijusPuras — APPROVED** [review]
looks good to me

File: Frontend/src/Services/ItemService.ts [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+    async upload(data: UploadData, navigate: NavigateFunction ) {
         return http.post<UploadData>(`${BASE_URL}/Item`, JSON.stringify(data)).then(
-            (res) => res.status === 200 ? toast.success("Item was successfully uploaded") : toast.error(`Error code: ${res.status}`)
+            (res) => res.status === 200 ? toast.success("Item was successfully uploaded", {
+                onClick: () => navigate(`/details/${res.data}`)
+            }) : toast.error(`Error code: ${res.status}`)
         );
     }
+    async getAll(props?: getAllProps) {
+        console.log(props)
```
[the remark this answers is not in the document]
Comment (somkamarius) [reply Frontend/src/Services/ItemService.ts]: thanks for catching these!

**Comment by somkamarius** [comment]
> Condition filter in BrowseScreen needs to be finished and getAll from ItemServices should be rewritten, since it breaks the single-responsibility principle

Yeah will adjust these in the future now. Added in Jira

**Pull request merged**